modules/travel_pay/app/models/travel_pay/mileage_expense.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module TravelPay
+  class MileageExpense < BaseExpense
+    VALID_TRIP_TYPES = %w[ONE_WAY ROUND_TRIP].freeze
+
+    attribute :trip_type, :string
+    attribute :requested_mileage, :float
+
+    validates :trip_type, presence: true, inclusion: { in: VALID_TRIP_TYPES }
+    validates :requested_mileage, numericality: { greater_than: 0.0 }, allow_nil: true
```

Comment:
Why `allow_nil`? Guardrail against missing data from the API?

Reply:
Oh. Right. This is a weird one. Probably fine for now, but I wonder if it makes sense at all to separate the request model from the response model... The `requested_mileage` field must be optional because we don't send it to the API, but the API sends it back.

Reply:
Yeah, I remember being curious when talking about it in grooming. If we had different models for request/response, do you think we'd only split them up like that for cases that had different validation requirements? Like, if one of the models _doesn't_ need different validation for the request vs. response, we'd just use the one?